src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickBrewing.kt

```diff
@@ -0,0 +1,34 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import net.minecraft.client.Minecraft
+import net.minecraft.client.gui.inventory.GuiChest
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class ShiftClickBrewing {
+
+    @SubscribeEvent
+    fun onSlotClick(event: GuiContainerEvent.SlotClickEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!SkyHanniMod.feature.inventory.shiftClickBrewing) return
+
+        if (event.gui !is GuiChest) return
+
+        if (event.slot == null) return
+
+        val chestName = InventoryUtils.openInventoryName()
+        if (!chestName.startsWith("Brewing Stand")) return
+
+        event.isCanceled = true
+        Minecraft.getMinecraft().playerController.windowClick(
+            event.container.windowId,
+            event.slot.slotNumber,
+            event.clickedButton,
+            1,
+            Minecraft.getMinecraft().thePlayer
+        )
```

Comment:
This potentially sends an invalid packet if `event.clickedButton == 2`. Please always send `0` as button. See https://wiki.vg/index.php?title=Protocol&oldid=7368#Click_Window for valid modes / buttons.

Reply:
thy. I didn't notice the illegale state. Thought shift MMB was a thing 😅